pkg/reconciler/managed/reconciler.go

```diff
@@ -29,6 +29,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
+	changelogs "github.com/crossplane/crossplane-runtime/apis/changelogs/proto/v1alpha1"
```

Comment:
```suggestion
	"github.com/crossplane/crossplane-runtime/apis/changelogs/proto/v1alpha1"
```

I think we normally import generated proto types in a similar way to Kubernetes types. The convention is that if it's a Crossplane type (as opposed to upstream Kubernetes or third party) we just import it using its version - so `v1alpha1`. If we end up with two different `v1alpha` packages then we go to e.g. `changelogsv1alpha1` to differentiate.